packages/kolibri-common/components/SearchFiltersPanel/index.vue

```diff
@@ -22,6 +22,10 @@
       :class="windowIsLarge ? '' : 'drawer-panel'"
     >
       <div v-if="!accordion">
+        <KLinearLoader
+          v-if="searchLoading"
+          class="linear-loader"
+        />
```

Comment:
This linear loader was added to indicate that a search was in progress or the search filter labels were loading on the `LibraryPage` on smaller screen sizes. The loader is set to 120% of its with so that it would be the full length of the outer container of the  `SearchFiltersPanel`:

https://github.com/user-attachments/assets/20a755c4-f0de-4c51-9dc9-33bae00a8fc5